### DISCUSSION TITLE
How to work with double and float variables?

### DISCUSSION THREAD

**Discussion by Shlepsss** [CosmosOS/Cosmos#2960] [Q&A] [answered]
when i trying to work with float or double type i get error. What i should do for normal float/double work?

**Comment by retrievaaaa** [comment]
What do you mean with "i get error"? What type of error - compile-time or runtime, what error specifically, what are you trying to do that causes that error, etc.

**Reply by Shlepsss** [reply]
I have a calculator app in my OS, and when I start calculator I see, that try{} catch{} construction catch an exception.

**Reply by blackneeed** [reply]
but what is the exception

**Reply by blackneeed** [reply]
when you start so runtime

**Reply by Shlepsss** [reply, marked as answer]
ouch, sorry, that was my error and double type working normally, i find a bug in calculator and fix it. Now all OK. Thanks for help)
